respondd.py

```diff
@@ -51,7 +51,9 @@ def handle(self):
                     response = str.encode(json.dumps(answer))
 
             if response:
-                socket.sendto(response, self.client_address)
+                addr, port, flow, scope = self.client_address
+                real_ifindex = socket.if_nametoindex(iface)
+                sock.sendto(response, (addr, port, 0, real_ifindex))
```

Comment:
`iface` is `util.ifindex_to_iface(ifindex)`, so you can just use `ifindex` instead of looking up the name (I just happened to make the same fix to the C respondd implementation, as I was looking into VRFs recently)